data/mods/Aftershock/items/comestibles/alienfood.json

```diff
@@ -132,5 +173,406 @@
+    "weight": "56 g",
+    "volume": "250 ml",
+    "color": "pink",
+    "charges": 4,
+    "spoils_in": "1 day",
+    "healthy": -2,
+    "price_postapoc": 25,
+    "quench": 2,
+    "fun": -10,
+    "calories": 50,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 9 ], [ "calcium", 0 ], [ "iron", 14 ], [ "vitB", 22 ], [ "mutant_toxin", 25 ] ],
+    "extend": { "flags": [ "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_lung_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal_cooked",
+    "name": { "str": "cooked piece of alien lung", "str_pl": "cooked pieces of alien lung" },
+    "description": "The organ no longer leaks out a strange liquid, and faded to a light blue color. It is chewy, bitter, and smells unpleasant.",
+    "parasites": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_liver",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien liver",
+    "description": "A porous hard organ. Hazarding a guess from its position in the body, this is the liver.",
+    "price_postapoc": 25,
+    "quench": -4,
+    "calories": 92,
+    "vitamins": [ [ "vitA", 75 ], [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 8 ], [ "vitB", 336 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_liver_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_liver",
+    "looks_like": "meat_cooked",
+    "name": "cooked alien liver",
+    "description": "After some time cooking, the pores of the organ have shrunk, and its much more softer, almost yogurt-like in its consistency.",
+    "parasites": 0,
+    "price_postapoc": 50,
+    "fun": -2,
+    "healthy": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_brain",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": { "str_sp": "raw alien brains" },
+    "description": "Yep, definitely a brain.",
+    "color": "light_gray",
+    "healthy": -15,
+    "price_postapoc": 10,
+    "quench": 10,
+    "fun": -20,
+    "parasites": 15,
+    "calories": 78,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 2 ], [ "calcium", 1 ], [ "iron", 8 ], [ "vitB", 54 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_brain_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_brain",
+    "looks_like": "offal_cooked",
+    "name": { "str_sp": "cooked alien brains" },
+    "description": "Fried and scrambled, this doesn't look the most appetizing, but its edible.",
+    "price_postapoc": 25,
+    "parasites": 0,
+    "healthy": -2,
+    "fun": -8,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_kidney",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien kidney",
+    "description": "You are guessing here, but this is the kidney of the animal. The ammonia-like smell was the biggest hint.",
```

Comment:
"this is the" -> "this is likely the". And why "ammonia-like" and not "ammonia"?